recipe-client-addon/lib/NormandyDriver.jsm

```diff
@@ -78,11 +78,11 @@ this.NormandyDriver = function(sandboxManager) {
         syncSetup: Preferences.isSet("services.sync.username"),
         syncDesktopDevices: Preferences.get("services.sync.clients.devices.desktop", 0),
         syncMobileDevices: Preferences.get("services.sync.clients.devices.mobile", 0),
-        syncTotalDevices: Preferences.get("services.sync.numClients", 0),
         plugins: {},
         doNotTrack: Preferences.get("privacy.donottrackheader.enabled", false),
         distribution: Preferences.get("distribution.id", "default"),
       };
+      appinfo.syncTotalDevices = appinfo.syncDesktopDevices + appinfo.syncMobileDevices;
```

Comment:
Modifying the "shape" of an object after first creation is sub-optimal. Can we include the key in `appinfo` right away, probably by just having `syncTotalDevices: null,`?